ugs-pendant/src/main/java/com/willwinder/universalgcodesender/pendantui/v1/model/Settings.java

```diff
@@ -0,0 +1,45 @@
+package com.willwinder.universalgcodesender.pendantui.v1.model;
```

Comment:
I'm a big fan of using Kotlin for POJOs, the equivalent Kotlin:
```kotlin
data class Settings(
  val jogFeedRate: double,
  val jogStepSizeXY: double,
  val preferredUnits: UnitUtils.Units,
  val jogStepSizeZ: double
)
```

However, Kotlin -> Java interop is not as clean as Java -> Kotlin, so leaving it the way you have it is probably for the best.